dalf/searxng
# Fix Grid layout in simple upgrade

## What does this PR do?

This PR does the following:

* remove most of the margins and width css elements around results and the sidebar
* add vars for the gap between results and sidebar as well as the header and footer color
* make the grid layout work correctly on mobile and tablet screens
* remove some stuff from the toolkit to make elements fit more cohesive in the UI (since now this stuff in controlled threw the grid layout anyway)



## Why is this change important?

Improves the grid layout change so it gets closer to being merged upstream...


## How to test locally

```make themes.all```

```make run```

* In firefox you can enable the grid layout view in the dev tools to check if it works correctly in conjunction with the responsive design mode

## Author's checklist

Desktop

![image](https://user-images.githubusercontent.com/38733246/134774758-c8ddc2e0-3470-4ed5-bb6a-c9e7ca05ed98.png)


Tablet

![image](https://user-images.githubusercontent.com/38733246/134774798-8776e415-6ab4-439d-b554-46d67702bcbf.png)


Phone

![image](https://user-images.githubusercontent.com/38733246/134774773-c676ac87-f860-461a-9f2a-830a3f0b7030.png)


## Related issues
